ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -289,6 +289,39 @@ def test_tensorflow_Asin(
     )
 
 
+# argmax
+@handle_cmd_line_args
+@given(
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric", full=True),
+        valid_axis=True,
+        force_int_axis=True,
+        min_num_dims=1,
+    ),
+    output_type=helpers.get_dtypes("numeric"),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.ArgMax"
+    ),
+)
+def test_tensorflow_ArgMax(
+    dtype_x_axis, as_variable, num_positional_args, fw, native_array, output_type
+):
+    dtype, x, axis = dtype_x_axis
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
```

Comment:
with_out should be passed in through the arguments of test_tensorflow_ArgMax as the @handle_cmd_line_args will fill in the with_out argument automatically.

Reply:
I fixed and pushed